dashboard/app/models/script.rb

```diff
@@ -333,7 +333,7 @@ def beta?
   end
 
   def self.beta?(name)
-    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5'
+    name == 'edit-code' || name == 'gradeKinder' || name == 'grade1' || name == 'grade2' || name == 'grade3' || name == 'grade4' || name == 'grade5' || name.start_with?('csd')
```

Comment:
Longer term it seems like these should really be properties on individual scripts instead of methods on the Script model. @joshlory is this something we've considered?

Also, not sure if we should be using the wildcard starts_with('csd'), since now someone can create a course and have it unexpectedly be in beta (because they started their course name with csd and didn't know this was a thing).